etc/cache.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<!--
+/**
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2015 Adyen BV (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Cache/etc/cache.xsd">
+    <type name="adyen" translate="label, description" instance="Adyen\Payment\Model\Cache\Type\AdyenCache">
+        <label>Adyen</label>
+        <description>Webhook cache</description>
```

Comment:
I would keep this description more generic, so that it can be used for other purposes
```suggestion
        <description>Adyen payment module cache</description>
```